modules/ROOT/partials/module-loading/bundling-plugin-files.adoc

```diff
@@ -124,6 +124,13 @@ include::partial$plugin-files/plugin-file-list-linkchecker.adoc[]
 
 include::partial$plugin-files/plugin-file-list-mentions.adoc[]
 
+[[mergetags]]
+=== Merge Tags (`+mergetags+`)
```

Comment:
This needs to be added to the list at the top of this page.

Reply:
Added xref to the ToC.